crates/bevy_pbr/src/light.rs

```diff
@@ -294,9 +294,26 @@ impl ClusterConfig {
             } => {
                 let aspect_ratio = screen_size.x as f32 / screen_size.y as f32;
                 let per_layer = *total as f32 / *z_slices as f32;
+                assert!(
+                    per_layer >= 1.0,
+                    "ClusterConfig has more z-slices than total clusters!"
+                );
```

Comment:
This catches a developer error so I think it should be a debug_assert so it gets compiled out for release builds.
```suggestion
                debug_assert!(
                    *total >= *z_slices,
                    "ClusterConfig has more z-slices than total clusters!"
                );
                let per_layer = *total as f32 / *z_slices as f32;
```